CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing
+
+Thank you for your interest in making it easier to write good CSS3! Here's how
+you can help!
+
+## Open an issue
+
+If you think something's missing, make sure you're not asking for something on
+[this list of bad CSS
+properties](https://gist.github.com/y0ssar1an/bb95223148e486acbe7a#file-
+bad_css). If it's not on that list, open an issue and we'll investigate. we'll
+be monitoring the specs as they're updated on [the W3C
+feed](http://www.w3.org/Style/CSS/current-work.en.html), but I still need your
+help. Let's keep bad code out of the Web!
+
+## Submit a pull request
+
+It would be awesome to have more people dive into these regular expressions with
+us! You should probably open an issue first before writing any code though.
+
+## Setup a dev environment
+
+If you're ready to dive into the code, then this is how. The following
+instructions are for hackers only.
+
+### Fork CSS3
+
+Create a fork of CSS3 on GitHub.
+
+### Install Sublime Text 3dev
```

Comment:
> We really can't release the new syntax format until the public build of ST3 supports it. So we can replace the part about using 3dev with a message that says "Make sure you're on Sublime Text 3 build XXXX or later". We'll put the same message in the release notes too (in `/messages`).

What if someone doesn't have build XXXX or later? How will they know where to get that build? We'll still have to have _something_ about 3dev, no?